runhouse/docker/slim/public-key-auth/instructions.md

```diff
@@ -0,0 +1,32 @@
+How to start a local Docker container with public key based authentication
+
+1. Configure Docker to use secrets in the build process
+
+echo "DOCKER_BUILDKIT=1" >> ~/.docker/config.json
```

Comment:
this didn't work for me so I edited the file manually (maybe bc I already had an existing dict saved to my config.json?)